Tests iOS/MyList/ArchiveTests.swift

```diff
@@ -144,9 +144,11 @@ class ArchiveTests: XCTestCase {
                 return Response.myList("unarchive")
             } else if apiRequest.isForMyListContent {
                 return Response.myList("list-with-unarchived-item")
+            } else if apiRequest.isForSlateLineup {
+                return Response.slateLineup()
+            } else {
+                fatalError("Unexpected request")
```

Comment:
I don't think you need to add these.  You're not testing that request in this test.

Reply:
Yeah this was temporary fix since the tests were failing, but didn't think I should be making this change so that's why I was investigating - but if you're not having issues without this change, could just be my faulty Xcode 🤔